### ISSUE TITLE
Crash: Auto-harvesting 

### ISSUE THREAD

**Issue by vbarkman** [devs-immortal/The-Aether#779]
### What were you doing when the crash occured? How can we replicate it?

1. Install a mod that allow auto harvest (i.e: Harvest with ease, or Amethyst imbuement, with the spell Abundance)
2. Plant flax and wait for it to mature,
3. Use the autoharvest feature of these mods on the flax,
4. The game freezes and requires manual shutdown, after restart the save freezes on reload and becomes functionnally corrupted.

### Did it crash the client or server?

Client

### Mod Version

2.1.0

### Fabric API Version

0.76.1+1.19.2

### Crash Report

```Shell
none, the game freezes with no noticeable additional log being generated.
```

### Other mods

Too big to upload, but It happened consistently with the autoharvesting mods mentioned above.

### Additional Information

-I'm using Modrinth

-Nothing else comes to mind, happy debugging.

![image](https://github.com/devs-immortal/Paradise-Lost/assets/9602872/6d466fd0-078c-4376-86d9-a8ae10411aa9)

**Comment by vbarkman** [comment]
My bad, it did actually generate log, but only when the client actually crashes which doesn't always happen, I attached it below:
[crash-2023-11-21_00.10.38-client.txt](https://github.com/devs-immortal/Paradise-Lost/files/13420166/crash-2023-11-21_00.10.38-client.txt)

**Comment by MBatt1** [comment]
I think this was fixed in 2.2.1 however I can't confirm and this issue is quite old. Will investigate if a newer bug report is opened thank you
